docs/user-guide/global-configurations/user-access.md

```diff
@@ -62,84 +108,85 @@ Select a project from the drop-down to which you want to give permission to the
 
 In the `Environment` section, you can select one or more than one or all environments at a time. Click on the environment section, you will see a drop-down of your environments and select any environment on which you want to give permission to the user.
 
+**`IMP`** If `all environments` option is selected then user gets access to all current environments and any new environment which gets associated with this application later.
+
 * **Applications**
 
 Similarly, you can select `Applications` from the drop-down corresponding to your selected Environments. In this section, you can also give permissions to one or more than one or to all applications at a time.
 
-* **Roles**
+**`IMP`** If `all applications` option is selected then user gets access to all current applications and any new application which gets associated with this project later.
 
-  Inside the `Role`, you actuallyhoose which type of permissions you want to give to the users.
+* **Roles**
 
-There are four different view access levels/Role available for both User and Group, namely:
+  Inside the `Role`, you actually hoose which type of permissions you want to give to the users.
 
-* **View Only**: User\(s\)/Group\(s\) can only view the selected applications.
-* **Build and Deploy**: User\(s\)/Group\(s\) under build & deploy also have `View Only` permissions but apart from the view only permission they have the permission to trigger CI/CD pipelines of the applications of selected environments.
-* **Admin**: User\(s\)/Group\(s\) who have `Admin` permission can view, trigger, and edit selected applications.
-* **Manager**: User\(s\)/Group\(s\) under manager also have the `Admin` permissions but apart from that they can create, edit, and delete the selected applications, can also manage `User access` but the user can only give permissions on those applications which the user has access to.
+There are four different view access levels/Role available for both User and Group as described [above](#access-levels):
 
-![](../../.gitbook/assets/gc-user-access-configure-direct-permission.jpg)
+![](../../.gitbook/assets/gc-user-access-configure-direct-permission%20%282%29.jpg)
 
 You can add multiple rows, for Direct Permissions.
 
 Once you have finished assigning the appropriate permissions for the listed users, Click on `Save`.
 
-Chart group permissions
+### Chart group permissions
 
-You can also manage the access of users to Chart Groups in your project. This is optional, if you want to give permission regarding your helm charts in that case you can use it. By default, all users have the `View` permission of the charts.
+You can also manage the access of users to Chart Groups in your project. This is optional, if you want to give permission for helm chart groups then it can be used. By default, all users have the `View` permission of the charts for the environments on which they have access to any application.
```

Comment:
please simplify this statement `This is optional, if you want to give permission for helm chart groups then it can be used`